src/views/Info/components/InfoCharts/CandleChart/index.tsx

```diff
@@ -101,16 +101,22 @@ const CandleChart = ({ data, setValue, setLabel, ...rest }: LineChartProps) => {
   useEffect(() => {
     if (chartCreated && data) {
       const series = chartCreated.addCandlestickSeries({
-        upColor: theme.colors.success,
-        downColor: theme.colors.failure,
-        borderDownColor: theme.colors.failure,
-        borderUpColor: theme.colors.success,
-        wickDownColor: theme.colors.failure,
-        wickUpColor: theme.colors.success,
+        upColor: '#31D0AA', // theme.colors.success has some issue
+        downColor: '#ED4B9E', // theme.colors.failure has some issue
+        borderDownColor: '#ED4B9E',
+        borderUpColor: '#31D0AA',
+        wickDownColor: '#ED4B9E',
+        wickUpColor: '#31D0AA',
```

Comment:
need to directly `import { baseColors } from '@pancakeswap/ui/tokens/colors'` to avoid using css var